### DISCUSSION TITLE
bug: changed windows highlight groups

### DISCUSSION THREAD

**Discussion by joellidin** [rose-pine/neovim#234] [Q&A] [answered]
### Neovim version (nvim -v)

NVIM v0.10.0-dev-1968+ge0eb4188b

### Terminal / multiplexer

tmux

### Describe the bug

So this is not really a bug, it is just that some highlights group seem to have changed and I can't figure out exactly which ones. So this is how windows looked for me before commit bf00df1 (in committia.vim):
![image](https://github.com/rose-pine/neovim/assets/19657866/321f7ef2-85e8-42f9-a27d-8e1fd994da9a)
After commit bf00df1 I instead have this:
![image](https://github.com/rose-pine/neovim/assets/19657866/6e3dfbc1-d60a-4516-9396-d4ed21bd3510)
Is there anyone that know which highlight groups I have to change in order to remove the background of the non active windows?

### Repro

```Lua
vim.o.packpath = "/tmp/nvim/site"

local plugins = {
  rose_pine = "https://github.com/rose-pine/neovim",
  -- ADD OTHER PLUGINS _NECESSARY_ TO REPRODUCE THE ISSUE
}

for name, url in pairs(plugins) do
  local install_path = "/tmp/nvim/site/pack/test/start/" .. name
  if vim.fn.isdirectory(install_path) == 0 then
    vim.fn.system({ "git", "clone", "--depth=1", url, install_path })
  end
end

require("rose-pine").setup({
  -- ADD ROSÉ PINE CONFIG THAT IS _NECESSARY_ TO REPRODUCE THE ISSUE
})

vim.cmd("colorscheme rose-pine")
```

**Comment by mvllow** [comment, marked as answer]
You may want to set `transparency = true`:

```lua
require("rose-pine").setup({
  styles = {
    transparency = true
  }
})
```

Otherwise it should be `Normal` or something specified by the plugin itself, which you could maybe find by tab completing `:hi PluginName<tab>`.

Edit: I noticed your cursor in the top left buffer so the ones with a background might actually be `NormalNC`. If you share your Rosé Pine config I can help further, but moving this to a discussion unless a bug is found :)

**Reply by joellidin** [reply]
Thank you very much. Adding your suggestions does work perfectly. And, the highlight group I was looking for was indeed the `NormalNC`. I appreciate the help and thank you for this great colorscheme.

**Comment by Mango0x45** [comment]
I’m having this issue too.  I made the change suggested by @mvllow which _works_, but it also makes the background of context- and popup-menus such as autocompletion buffers and the right-click menu transparent.  How can I avoid this?

**Reply by mvllow** [reply]
You could either override any groups you want transparent without actually using the `transparency` style, or you could "undo" any styles from `transparency` that don't fit your needs, e.g.:

```lua
{
  styles = {
    transparency = true,
  },
  highlight_groups = {
    Pmenu = { bg = "surface" },
  }
}
```

I've not tested the above but if either suggestion isn't working, please share your Rosé Pine config and we can try to help more!

**Reply by Mango0x45** [reply]
That was helpful, thanks!
